src/components/charts/SummaryChartGroup/index.jsx

```diff
@@ -80,6 +83,7 @@ class SummaryChartGroup extends Component {
 
 SummaryChartGroup.propTypes = {
   summaries: PropTypes.arrayOf(PropTypes.object).isRequired,
+  totalCount: PropTypes.number.isRequired,
```

Comment:
I'm in favor of making this new props an optional one, if omitted, use old logic to calculate percentage